### PR TITLE
Add Italian translation

### PR DESCRIPTION
**Pull request by alef**
**When merged this pull request will:**
- Add Italian translation for most CBA components (9 left out)
- Respect the [Submitting Content Guidelines](https://github.com/CBATeam/CBA_A3/wiki/Submitting%20content)

### CODE REVIEW

File: addons/strings/stringtable.xml [hunk 1]
```diff
@@ -30,13 +31,8 @@
         </Key>
         <Key ID="STR_CBA_FORMAT_NUMBER_THOUSANDS_SEPARATOR">
             <English>,</English>
-            <Czech> </Czech>
```
Comment (PabstMirror) [inline addons/strings/stringtable.xml]: don't remove these

Comment (veteran29) [reply addons/strings/stringtable.xml]: Does arma support non breaking space character? I wonder if that could be used here, it's really easy to mess that when using Tabler.

File: addons/vectors/stringtable.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (commy2) [inline addons/vectors/stringtable.xml]: Which of these is right?

Comment (alef) [reply addons/vectors/stringtable.xml]: It seems Tabler saves _stringtable.xml_ files without [BOM](https://en.wikipedia.org/wiki/Byte_order_mark). On your master branch, only vectors/ , versioning/ and xeh/ have BOM. All the others do not. I would assume that the one without BOM (my branch) are right.

File: addons/strings/stringtable.xml (hunk 1 above)
Comment (bux) [reply addons/strings/stringtable.xml]: How would I differentiate between a user error and intentional blank space :/

File: addons/vectors/stringtable.xml (hunk 2 above)
Comment (dedmen) [reply addons/vectors/stringtable.xml]: I already mentioned somewhere that these 3 had invalid bom in the past. We also had a couple script files with invalid I think.

Comment (dedmen) [reply addons/vectors/stringtable.xml]: On master branch backpacks module stringtable still has bom.

**Pull request merged**